Makefile

```diff
@@ -0,0 +1,11 @@
+PROJECTNAME=$(shell basename "$(PWD)")
```

Comment:
@zullin with the go code on the root folder, how do we create two containers(one for web, one for api) for the application?